src/vivarium/framework/components/manager.py

```diff
@@ -193,9 +200,12 @@ def get_components_by_type(
             A list of components of type ``component_type``.
         """
         # Convert component_type to a tuple for isinstance
-        return [c for c in self._components if isinstance(c, tuple(component_type))]
+        component_type = (
```

Comment:
Was this required or is it adding functionality? This is less readable to me than the original.

Reply:
The previous functionality was incorrect, as you can only call `tuple()` on an iterable.